### ISSUE TITLE
WebpackError: error compiling the html.js component for the development server. 

### ISSUE THREAD

**Issue by phoolto** [justinwhall/gatsby-wordpress-netlify-starter#2]
I have tried to update the outdated dependencies but I cannot resolve this problem. Also reconfigured everything but I keep getting the following compiling error. 

at Object.<anonymous> (/Users/user/Documents/Gits/gatsby-wordpress-netlify-starter/node_modules/babel-loader/lib/index.js:82:26)
error There was an error compiling the html.js component for the development server.

See our docs page on debugging HTML builds for help https://goo.gl/yL9lND

  30 |  	__webpack_require__.m = modules;
  31 | 
> 32 |  	// expose the module cache
     | ^
  33 |  	__webpack_require__.c = installedModules;
  34 | 
  35 |  	// define getter function for harmony exports

  WebpackError: ./.cache/develop-static-entry.js

**Comment by justinwhall** [comment]
Hey there. Tough to say form the limited information above. 

I can say, that you can not just update dependancies and expect everything to work ( or even be supported ). This repo is Gatsby 1 for instance. Have you tried _not_ updating dependencies?

**Comment by phoolto** [comment]
I tried updating because it gave that error, updating however didn't change that. I already got the error without modifying anything. 

Now I figured out that babel-loader was missing in the package.json and I installed that one in the -d but now it outputs the following:

Module build failed (from ./node_modules/gatsby/dist/utils/babel-loader.js). Error: Plugin/Preset files are not allowed to export objects, only functions. In /Users/user/Documents/Gits/gatsby-wordpress-netlify-starter/node_modules/node_modules/babel-preset-react/lib/index.js
